commit-status-publisher-server/src/main/java/jetbrains/buildServer/commitPublisher/github/api/impl/GitHubApiImpl.java

```diff
@@ -164,6 +164,29 @@ public String findPullRequestCommit(@NotNull String repoOwner,
   }
 
   @Nullable
+  public String findPullRequestLabel(@NotNull String repoOwner,
+                                     @NotNull String repoName,
+                                     @NotNull String branchName) throws IOException {
+    final String pullRequestId = getPullRequestId(repoName, branchName);
+    if (pullRequestId == null) return null;
+
+    //  /repos/:owner/:repo/pulls/:number
+
+    final String requestUrl = myUrls.getPullRequestInfo(repoOwner, repoName, pullRequestId);
+    final HttpGet get = new HttpGet(requestUrl);
+    includeAuthentication(get);
+    setDefaultHeaders(get);
+
+    final PullRequestInfo pullRequestInfo = processResponse(get, PullRequestInfo.class);
+
+    final RepoInfo head = pullRequestInfo.head;
+    if (head != null) {
+      return head.ref;

```

Comment:
We should declare a variable to hold return value, and using **return** at the end of method only